.azure-pipelines/client.yml

```diff
@@ -76,17 +76,34 @@ jobs:
         Linux:
           OSName: 'Linux'
           OSVmImage: 'ubuntu-16.04'
+          VerifyOSScript: 'IsLinux.py'
         Windows:
           OSName: 'Windows'
           OSVmImage: 'vs2017-win2016'
+          VerifyOSScript: 'IsWindows.py'
         MacOs:
           OSName: 'MacOS'
           OSVmImage: 'macOS-10.13'
+          VerifyOSScript: 'IsMacOs.py'
```

Comment:
Could we not just pass in OSName as a parameter to a common script to avoid needing to add another variable to all our matrix combinations?